eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -126,6 +128,7 @@ stages:
             variables:
               - template: ../variables/globals.yml
             pool:
+              name: Azure Pipelines
               vmImage: vs2017-win2016
```

Comment:
Are we sticking with this older image because of the gpg signing still?

Reply:
Yep. I don't think it is worth spinning up another pool for this image. I'd like to get off it though (we use it in a few places).

Reply:
Yeah I'd like to move to the latest windows image to test this out again, but that can be done in another set of changes as I believe you still have an issue tracking it.